setup.py

```diff
@@ -133,6 +129,6 @@ def read(fname):
     extras_require={
         'web_extra': web_extra_requires,
         'hub': hub_requires,
-        'dev': web_extra_requires + hub_requires + dev_requires + docs_requires
+        'dev': web_extra_requires + hub_requires + dev_requires + docs_requires,
```

Comment:
is adding this trailing comma a preferred way now? Remember it was recommended to remove it, maybe this rule changed now.